### PR TITLE
Add recently merged tests and fix idempotence of db/seed.rb

### PR DESCRIPTION
**Pull request by seancribbs**
This one's for @joedevivo

### CODE REVIEW

File: db/seed.rb [hunk 1]
```diff
@@ -72,18 +73,21 @@ def create_riak_test(name, *args)
...
 ## Test upgrades on only persistent backends, from two different versions
 platforms.each do |p|
-  %w{bitcask eleveldb}.each do |b|
-    %w{previous legacy}.each do |v|
+  %w{previous legacy}.each do |v|
+    %w{bitcask eleveldb}.each do |b|
       create_riak_test "loaded_upgrade", 'platform' => p, 'backend' => b, 'upgrade_version' => v
-      create_riak_test "verify_basic_upgrade", 'platform' => p, 'backend' => b, 'upgrade_version' => v
     end
+    create_riak_test "verify_basic_upgrade", 'platform' => p, 'upgrade_version' => v
```
Comment (joedevivo) [inline db/seed.rb:89]: do you want to call out bitcask explicitly here?

Comment (seancribbs) [inline db/seed.rb:89]: If we want bitcask explicitly we should make a migration to fix those tests. As it stands, there is no backend specified in the current matrix.

**Comment by seancribbs** [comment]
The sad fact is, this obscures the fact that what we really need is a UI for adding new tests to the suites.

**Comment by cmeiklejohn** [comment]
I can work on that if you file an issue, once I wrap up what I'm currently working on.

**Comment by seancribbs** [comment]
@cmeiklejohn happy to talk it over with you, I have a few ideas

**Comment by joedevivo** [comment]
:turtle: sorry this took so long +1

**Pull request merged**